source/content/guides/drush/02-drush-versions.md

```diff
@@ -12,53 +12,65 @@ cms: [drupal]
 audience: [development]
 product: [--]
 integration: [drush]
+
 ---
 
-This section provides information on Drush versions and site-local usage.
 
-Pantheon runs Drush versions based on your Drupal version:
+## Understanding Drush installation
 
-- **Drupal (Latest Version)**: Drush 10 (for newly created sites)
-- **Drupal 7**: Drush 8 (for newly created sites)
+Drush is installed in two places on within the infrastructure of a modern Drupal site on Pantheon:
 
-## Available Drush Versions
+* **The global installation** is available to all sites and it's version number is set by the `pantheon.yml` file. When left unset in `pantheon.yml`, the version number will come from `pantheon.upstream.yml` or falls back to a default value.
+* **The site-local** installation is the one that is managed by a site's Composer files (`composer.json` and `composer.lock`) in the root of the site's git repository.
 
-<Partial file="drush-supported.md" />
+For older versions of Drupal (7 and earlier), the global installation of Drush is the only one available.
+For Drupal 8 and later, global invocation of Drush will look for a site-local version within the directory structure of the site.
+Allowing for different versions of Drush between the global and site-local level is valuable when many sites are present within the same runtime environment.
 
-We recommend managing your site through Composer. Visit the [Build Tools Workflow](/guides/build-tools/) for information on how to use Composer to manage Drupal sites on Pantheon, or the [Convert to Composer](/guides/composer-convert) guide to convert an existing site to a Composer-managed site.
+Pantheon's containerized infrastructure makes this distinction more academic than practical for interactions on the platform.
+This separation between global and site-local installations relevant when working with local development environments or other platform providers.
 
-## Verify Current Drush Version
+**For most teams running a modern version of Drupal, the site-local version of Drush is only one to pay attention to.**
+The [`composer.json` file used by newly created Drupal 9 and 10 sites](https://github.com/pantheon-upstreams/drupal-composer-managed/blob/main/composer.json) is set to allow Drush 11 or 12 as the site-local version.
 
-You can use [Terminus](/terminus/) to verify the current version of Drush running on your Pantheon site:
+### Changing the site-local Drush version
 
-```bash{promptUser: user}
-terminus drush <site>.<env> -- status | grep "Drush version"
+To change the site-local version of Drush run a composer command to update the version of Drush in the `composer.json` file. For instance, to set the site-local version of Drush to 11, run the following command:
+
+```bash
+composer require drush/drush:^11
 ```
 
-## Compatibility and Requirements
+### Changing the global Drush version on Pantheon
```

Comment:
I would add a note somewhere that says something like "For modern versions of drupal, we strongly recommend using the local installation method."